cli/install/template.go

```diff
@@ -1,46 +1,52 @@
 package install
 
-// Template provides the base template for the `linkerd install` command.
-const Template = `{{ if not .SingleNamespace -}}
+// Chart provides the Chart.yaml file needed to validate chart metadata
+var Chart = []byte(`apiVersion: "v1"
+name: "linkerd"
+version: 0.1.0`)
```

Comment:
Yep I can add in `appVersion` for this. I can also add a few additional chart values that would be helpful since this should not just be a _minimal_ working chart.